docfx.json

```diff
@@ -9,7 +9,9 @@
         "exclude": [
           "**/toc.yml",
           "overwrites/**/*",
```

Comment:
- Exclude picking up the document markdown in the repo at api/overview/azure by adding it to exclude glob. 
- Added a write of toc.yml output _per moniker_ as output from the openpublishingConfig.
- Copying out specific folders so that we can have documents at the same url but with different content.